diagnostic.md

```diff
@@ -39,31 +36,22 @@ methods.
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is the place where all the action takes place. It acts as the middleman
+or translator between javascript and the different web browsers
 
 ## Question 6
 
 Please give two examples of things that are commonly confused for the DOM.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is commonly confused to be the view source and the html code that we write.
 
 ## Question 7
 
 Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
```

Comment:
`$('div').on('click', someDefinedFunction());`
You can use either `.on` or `.click`.